contracts/oracle/src/contract/alarms.rs

```diff
@@ -25,14 +26,23 @@ use crate::{
 
 pub type AlarmReplyId = u64;
 
-pub struct MarketAlarms;
+pub struct MarketAlarms(AlarmReplyId);
```

Comment:
[probably out of the scope of this PR] parameterizing by BaseC?

Reply:
Discussed offline that the untyped `::remove` on alarms reply is an obstacle. @go2bodhi please file an issue and link it here.

Reply:
New related issues were added: #47 , #48